wemake_python_styleguide/violations/consistency.py

```diff
@@ -425,3 +427,35 @@ class BadNumberSuffixViolation(TokenizeViolation):
     #: Error message shown to the user.
     error_template = 'Found underscored number: {0}'
     code = 310
+
+
+class MultipleInComparisonViolation(ASTViolation):
+    """
+    Forbids comparision where multiple 'in's are userd in a statement.
+
+    Reasoning:
+        This is unreadable. Use different comparisons for it.
+        Bring a consistency to the comparison!
+
+    Solution:
+        Refactor your comparison expression.
```

Comment:
Cab you please extend your idea here? Something like: "Refactor your comparison expression to use several ``and`` conditions or separate ``if`` statements in case it is appropriate."

Reply:
Sure